Include/cpython/pystate.h

```diff
@@ -176,8 +176,24 @@ struct _ts {
     PyObject **datastack_limit;
     /* XXX signal handlers should also be here */
 
+    // This must be last.
+    struct {
+        int initialized;
```

Comment:
Use a `char` or even a `bool` now that PEP 7 allows it.

Reply:
will do

FWIW, I had started using bool in other PRs a while back but backed off when there was a little resistance.  However, I'll doing it going forward.

Reply:
Other than this, do you have any objections to this PR.  I'd like to merge it today if possible.

Reply:
What about 
https://github.com/python/cpython/pull/29883#discussion_r77057181
and
https://github.com/python/cpython/pull/29883#discussion_r770573238
?

Reply:
I'm dropping "initialized". It isn't needed any more since the simplification I did earlier.